pkgs/development/octave-modules/fits/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildOctavePackage
+, lib
+, fetchurl
+, cfitsio
+}:
+
+buildOctavePackage rec {
+  pname = "fits";
+  version = "1.0.7";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/octave/${pname}-${version}.tar.gz";
+    sha256 = "0jab5wmrpifqphmrfkqcyrlpc0h4y4m735yc3avqqjajz1rl24lm";
+  };
+
+  nativeBuildInputs = [
+    cfitsio
+  ];
+
+  meta = with lib; {
+    homepage = "https://octave.sourceforge.io/fits/index.html";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ KarlJoad ];
+    description = "Functions for reading, and writing FITS (Flexible Image Transport System) files using cfitsio";
+    # Marked this way until KarlJoad gets cfitsio as a runtime dependency.
```

Comment:
In the initial commits, I just got all the packages into Nix. During that time, I didn't have a robust way of giving runtime-dependencies needed by packages to Octave. Now that I do, I have a branch (not in this PR, yet) where I fix up most of the packages that have runtime dependencies and remove the `broken = true;` flag. I am still working out some kinks in those before I pull them (and squash them) back into here.

There are still some packages that are still marked with `broken = true` because their runtime dependency doesn't exist in nixpkgs yet.